src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -1538,6 +1542,7 @@ class InputsEmbedderPhi3V : public InputsEmbedder::IInputsEmbedder {
         features_length += tokens.back().get_shape().at(1);
         ov::Tensor inputs_embeds{ov::element::f32, {1, features_length, m_vlm_config.hidden_size}};
         size_t offset = 0;
+        m_embedding.set_cpu_out_tensor();
```

Comment:
PR description says RemoteTensor, while here CPU tensor is called. How to understand it?

Reply:
The other fiction is called remote. CPU is to reset it back

Reply:
maybe we can pass argument to `m_embedding.infer` to denote which tensor type we expect?